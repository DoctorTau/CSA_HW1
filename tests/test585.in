534
6103 4950 5013 -3343 -3064 7246 7544 2289 -1282 -2973 6197 6495 -2357 4726 -2705 9999 -2545 9322 -1009 5792 3479 -9841 2112 7891 -4072 -646 3281 1287 -2024 -7564 -8397 3491 -1513 -1592 6650 -4383 -7989 -4287 -6068 8873 -385 9106 9775 -5423 3884 1872 -5774 -848 4059 1879 -8890 -9680 -1977 -9048 6901 -1662 -2144 -401 1648 6218 -8287 -7425 -6433 9774 793 -6196 5324 6156 -2658 -9319 82 -5398 1936 -9315 1577 9829 -9713 5225 1163 -3978 -650 1096 5081 -8757 -5360 -6505 1051 707 -9655 730 1732 -4688 7614 -8917 879 -1669 -342 1227 7468 -973 7626 3409 -8910 -5567 9732 7109 9193 -547 -8652 -6060 -4709 -3820 8103 -9616 6306 6682 4907 3507 3000 -9749 9262 5780 -7228 -8212 -5372 -1181 5421 6989 -8778 -4642 -1022 8473 -7304 7508 -1051 -1201 8907 -3811 -7342 -4620 -6704 -6126 1370 -9208 -2741 9907 -3527 8714 2533 -1448 -1439 -1101 -6839 -9584 -3683 -134 -1057 567 7368 5710 -2029 -8126 -5115 -2790 4802 1102 -9176 3261 236 -6288 -2123 -8073 -4991 496 -2965 1193 5894 -2698 8693 -531 -3095 -8968 9311 6815 2288 -9441 -5888 4241 5460 -5212 2813 -8395 9146 4323 8346 2604 -5633 -1903 1916 -2441 -752 -9166 -7825 -135 -9445 -2126 6658 9155 4788 2754 4650 3204 158 8085 8870 7387 -7831 8504 -1200 -6166 2481 2415 2544 -6175 7711 -1699 -9502 169 2381 8754 2093 -1486 -1604 -6871 -4904 -9949 -3125 7007 1914 4117 5788 8545 7910 7288 -9445 7063 596 1831 -2458 9804 6350 -7570 4552 2712 -2388 6601 7671 9655 -4748 541 -9327 -5774 7706 -3330 -1388 3466 -1374 -4606 -8171 8078 8903 -1646 3058 -4798 -8221 -7501 -1886 -9009 -194 -7038 -182 -6418 -4448 -4193 9030 9155 -4332 3202 -3546 -1867 -351 -8655 6944 -1067 9167 3164 1209 6975 7930 4936 -4062 1836 -6729 -9657 7858 7317 612 -4258 -6265 4499 98 -1045 -69 -972 2754 7611 5774 -2866 3736 -7356 -3802 -5535 7604 -5709 7939 5069 -3468 -3279 -9681 9722 -2548 -777 6546 8871 3214 5591 7500 8984 -1490 6703 8784 9065 -4660 -154 7118 4178 2306 2489 -648 8213 -7976 2264 -7467 8873 8336 -9037 -4051 7866 1541 9677 -2257 1359 -1139 -2203 -7925 3491 5836 -6531 1286 -5020 -8223 -1335 300 -129 7356 58 2359 5407 -437 -8812 2995 2857 -7868 9388 -5967 2933 -7012 -9209 924 -2702 4177 -9683 4142 -1173 -3248 6512 -7899 -1694 -4287 4324 -3874 3306 189 7867 6445 -9404 4103 -7671 -969 8402 6304 -8693 -3142 -2613 -4302 -3263 8238 -5236 -3953 -4851 -9049 -137 -3135 -1346 -4213 -9014 -7456 -7417 5572 6678 -3504 -4952 -9564 4883 -4514 -979 1231 1011 1661 4092 9167 -3269 -1115 -8630 -611 1561 5015 7793 7982 675 8622 89 3754 9138 -8448 9031 -6523 9897 1091 3687 -5253 9179 5753 -2332 -7336 9581 2147 2790 6070 -7687 -4214 -3195 6137 -5185 -9610 -7680 -8808 9187 9355 -3785 -1924 6852 250 -3770 -5509 -2573 -7409 2980 8789 -9367 3129 -1173 -2831 4188 -2348 7992 -5548 3193 -285 4455 -9082 -2305 -928 7185 1304 -3710 6477 -7412 -7810 9060 -5676 241 -7498 4615 -1808 5084 8652 -2789 4371 -1152 899 -1636 8663 -1877 -970 -6309 8415 5719 832 1209 1371 6805 -5907 -4708 