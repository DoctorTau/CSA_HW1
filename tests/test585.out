-3064 -1282 -4072 -646 -2024 -7564 -1592 -6068 -5774 -848 -8890 -9680 -9048 -1662 -2144 -6196 -2658 -5398 -3978 -650 -5360 -4688 -342 -8910 -8652 -6060 -3820 -9616 -7228 -8212 -5372 -8778 -4642 -1022 -7304 -7342 -4620 -6704 -6126 -9208 -1448 -9584 -134 -8126 -2790 -9176 -6288 -2698 -8968 -5888 -5212 -752 -9166 -2126 -1200 -6166 -9502 -1486 -1604 -4904 -2458 -7570 -2388 -4748 -5774 -3330 -1388 -1374 -4606 -1646 -4798 -1886 -194 -7038 -182 -6418 -4448 -4332 -3546 -4062 -4258 -972 -2866 -7356 -3802 -3468 -2548 -1490 -4660 -154 -648 -7976 -5020 -8812 -7868 -7012 -2702 -3248 -1694 -3874 -9404 -3142 -4302 -5236 -1346 -9014 -7456 -3504 -4952 -9564 -4514 -8630 -8448 -2332 -7336 -4214 -9610 -7680 -8808 -1924 -3770 -2348 -5548 -9082 -928 -3710 -7412 -7810 -5676 -7498 -1808 -1152 -1636 -970 -4708 